rero_mef/places/idref/mappings/v7/places_idref/idref-place-v0.0.1.json

```diff
@@ -0,0 +1,136 @@
+{
+  "settings": {
+    "number_of_shards": 8,
+    "number_of_replicas": 2,
+    "max_result_window": 100000000
+  },
+  "mappings": {
+    "date_detection": false,
+    "numeric_detection": false,
+    "properties": {
+      "$schema": {
+        "type": "keyword"
+      },
+      "pid": {
+        "type": "keyword"
+      },
+      "type": {
+        "type": "keyword"
+      },
+      "md5": {
+        "type": "keyword"
+      },
+      "identifiedBy": {
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "keyword"
+          },
+          "source": {
+            "type": "keyword"
+          },
+          "value": {
+            "type": "keyword"
+          }
+        }
+      },
+      "bnf_type": {
+        "type": "text"
+      },
```

Comment:
Do we really need/use this field? It is useful for concepts to differentiate subjects from genre/form but places will always be subjects

Reply:
The structure is more or less like in concepts, thats way I have not changed this.